segmentation_labeling_app/transforms/array_utils.py

```diff
@@ -44,6 +44,89 @@ def content_boundary_2d(arr: Union[np.ndarray, coo_matrix]) -> np.ndarray:
     return top_bound, bot_bound, left_bound, right_bound
 
 
+def content_extents(
```

Comment:
What differentiates this from `center_pad_2d`?

Reply:
`center_pad_2d` pads an array, for example a cropped ROI, with zeros. But, I want to know from this the indices that would crop it out of the original video frame. You can see its use in `TransformPipeline`

Reply:
I copied in the images (see PR description comment above) from PR #34 (now absorbed into this one). You can't index a movie with `center_pad_2d` the way I wanted to in the demo.

Reply:
Hm, I see. So you want `content_boundary_2d` but to work for the 3d array?

Reply:
I was just looking for places to consolidate since I see a lot of similar looking code. So if we can reuse more then it would clean it up a bit. I see now it's more of a nit.

Reply:
yes, that's right.
I agree, the two are very similar and could be combined. That was why I had said "code not optimal" in the original PR. But, these are touchy corner case things and didn't want to mess with this too much right now.